mozci/push.py

```diff
@@ -1042,6 +1047,13 @@ def get_regressions(
                             if (
                                 summary.status != Status.PASS
                                 and not summary.is_intermittent
+                                and (
+                                    not assume_unclassified_is_intermittent
+                                    or all(
+                                        c != "not classified"
+                                        for c, n in summary.classifications
+                                    )
```

Comment:
Thank you much for the reviews.
I had to change this condition because of the way tasks are stored as "runnable_summaries". I don't fully understand this part, but it make sense according to the module and tests.

Reply:
Basically, "runnable" is an abstraction. We have both "groups of tests" and "tasks", and "runnable" is the abstraction for them.